forecasttools/recode_locations.py

```diff
@@ -0,0 +1,23 @@
+import polars as pl
+
+import forecasttools
+
+
+def loc_abbr_to_flusight_code(
+    df: pl.DataFrame, location_col: str
+) -> pl.DataFrame:
+    # get location table
+    loc_table = forecasttools.location_table
+    # map abbr name to codes
+    mapping_dict = dict(
+        zip(loc_table["short_name"], loc_table["location_code"])
+    )
+    # replace column values with codes
```

Comment:
Not so familiar with polars, but it appears this is what [polars.Expr.map_dict](https://docs.pola.rs/api/python/version/0.18/reference/expressions/api/polars.Expr.map_dict.html) is for.